source/nogan/nogan.js

```diff
@@ -1266,11 +1266,18 @@ export const refresh = (
 	{ snapshot = getClone(nogan), past = [], future = [], operate = true, filter } = {},
 ) => {
 	const operations = []
-	const memo = new GetPeakMemo()
+	// const memo = new GetPeakMemo()
 	for (const id of iterateCellIds(snapshot)) {
 		if (filter && !filter(id)) continue
 		for (const colour of PULSE_COLOURS) {
-			const peak = getPeak(snapshot, { id, colour, past, future, memo })
+			const peak = getPeak(snapshot, {
+				id,
+				colour,
+				past,
+				future,
+				// TODO: restore memo here. it was for perf but was causing issue
+				// memo,
+			})
```

Comment:
Sad times, this was causing issues, but it was also causing big perf boosts. It *shouldn't* cause that issue, so there must be some bigger issue